src/LodeRunner.API.Test/IntegrationTests/Controllers/LoadTestConfigs.cs

```diff
@@ -105,6 +105,21 @@ public async Task CanCreateAndGetLoadTestConfigById()
             await httpClient.DeleteItemById<LoadTestConfig>(LoadTestConfigsUri, gottenTestRun.Id, this.output);
         }
 
+        /// <summary>
+        /// Determines whether this instance [returns correct response when it fails get a test run by invalid ID].
+        /// </summary>
+        /// <returns><see cref="Task"/> representing the asynchronous integration test.</returns>
+        [Fact]
+        [Trait("Category", "Integration")]
+        public async Task CannotGetLoadTestConfigByInvalidId()
+        {
+            using var httpClient = ComponentsFactory.CreateLodeRunnerAPIHttpClient(this.factory);
+
+            string invalidId = "xxxx-0000";
```

Comment:
I see this var in several places.  Does it make sense to make invalidId a class scoped variable?  Move it to the class declarations as a `private const string` maybe?  

Also, should it be named differently for different types of Ids, e.g., LoadTestConfigBadId, or is is the same invalid Id used regardless of the object type?

Reply:
Moved invalidId to class declarations and renamed it.